.dockerignore

```diff
@@ -0,0 +1,22 @@
+playwright/__pycache__/
```

Comment:
I'm afraid this `dockerignore` will rot over time. 

Instead of having one, you can put `Dockerfile` in an empty directory, e.g. `docker/Dockerfile.focal`, and use the directory as a docker context when building.

Reply:
Then its not possible to include files outside of the build context afaik.